src/api/dependency-discovery/jest.config.js

```diff
@@ -0,0 +1,9 @@
+const baseConfig = require('../../../jest.config.base');
+
+module.exports = {
+  ...baseConfig,
+  setupFiles: ['<rootDir>/src/api/dependency-discovery/jest.setup.js'],
```

Comment:
Include the root-level `jest.setup.js` too.

Reply:
Is it this file https://github.com/Seneca-CDOT/telescope/blob/master/test/jest.setup.js?

Reply:
Yes.  @rclee91 and @JiaHua-Zou had to deal with something similar in other tests where we didn't do this.  They can guide you.